src/main/resources/application.yml

```diff
@@ -51,7 +51,6 @@ chain:
   id: ${IEXEC_CHAIN_ID:17}
   sidechain: ${IEXEC_IS_SIDECHAIN:false}
   privateAddress: ${IEXEC_PRIVATE_CHAIN_ADDRESS:http://localhost:8545}
-  publicAddress: ${IEXEC_PUBLIC_CHAIN_ADDRESS:http://localhost:8545}
   hubAddress: ${IEXEC_HUB_ADDRESS:0xBF6B2B07e47326B7c8bfCb4A5460bef9f0Fd2002}
```

Comment:
In `ChainConfig`, `chainId`, `isSidechain` and `hubAddress` properties are read on `blockchainAdapterService`.
Should those properties be removed from `application.yml` and `README.md` ?

Reply:
We should do so, right.

Reply:
Better yes, see https://github.com/iExecBlockchainComputing/iexec-core/pull/546/commits/e6ec09d1b2d3d658dad17149a7e7b4684d239072